app/src/main/res/layout/app_version_fragment.xml

```diff
@@ -20,10 +20,11 @@
       <TextView
         android:id="@+id/app_version_text_view"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
+        android:layout_height="56dp"
```

Comment:
Do not set height it should be `wrap_content` only.

Reply:
You need following attributes:
size - 16sp
marginStart - 16dp
marginEnd - 16dp
marginTop - 20dp
marginBottom - 20dp

No padding at all.

color should be `oppiaPrimaryTextDark`

font should be `sans-serif`.

This needs to be done in both landscape and portrait mode.

Reply:
https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/screen/79e0846d-de5c-42b7-93aa-f93d4995616d/AC-AV-App-Version-/
if we observe  the text background is white and remaining is #E5E5E5 so to have that we should add background to FrameLayout 
if we keep the above attributes that suggested it will turn out to be like this!!
![WhatsApp Image 2020-03-13 at 18 06 09](https://user-images.githubusercontent.com/44704035/76621443-5de2ed80-6555-11ea-99d1-029b5690a564.jpeg)
if we fix text height to 56 and use padding it will turn out to be this!!
![WhatsApp Image 2020-03-13 at 18 09 57](https://user-images.githubusercontent.com/44704035/76621635-e6618e00-6555-11ea-933b-1c9767ee8ea6.jpeg)

Font and color of text are already set according to mocks!!
I will change the shadow to 1 dp  in my next PR
thankyou

Reply:
@chrk2205 Keep fixed heights for textview is a bad practice thats why we need wrap_content only. Now regarding background issue, if we change the margin to padding that I had mentioned earlier it would fix the issue for textview.

Meaning

paddingStart - 16dp
paddingEnd - 16dp
paddingTop - 20dp
paddingBottom - 20dp

No margin at all.

Also, the high-fi UI should be finished in a single PR and therefore actually you will need to introduce shadow for text view in this PR itself.

Reply:
Removed margins
All paddings are given accordingly
Used margins for app_last_update_date_text_view
background color given

thankyou